### PR TITLE
Fix issue no 329 and 330.

### PR DESCRIPTION
**Pull request by amitrnec**

### CODE REVIEW

File: Commons/src/main/java/eu/neclab/ngsildbroker/commons/controllers/EntryControllerFunctions.java [hunk 1]
```diff
@@ -379,7 +380,7 @@ public static ResponseEntity<String> upsertMultiple(EntryCRUDService entityServi
 			}
 		} else {
 			if (insertedOneEntity && appendedOneEntity) {
-				status = HttpStatus.MULTI_STATUS;
+				status = HttpStatus.CREATED;
```
Comment (ScorpioBroker) [inline Commons/src/main/java/eu/neclab/ngsildbroker/commons/controllers/EntryControllerFunctions.java:383]: This should be multi status. According to 6.15.3

File: Commons/src/main/java/eu/neclab/ngsildbroker/commons/controllers/EntryControllerFunctions.java [hunk 2]
```diff
@@ -188,8 +188,9 @@ private static ResponseEntity<String> generateBatchResultReply(BatchResult resul
 		if (result.getSuccess().isEmpty()) {
 			status = HttpStatus.BAD_REQUEST;
 		}
-		if (body == null) {
-			return ResponseEntity.status(status).build();
+		if (result.getFails().isEmpty() && !result.getSuccess().isEmpty()) {
```
Comment (ScorpioBroker) [inline Commons/src/main/java/eu/neclab/ngsildbroker/commons/controllers/EntryControllerFunctions.java:191]: Only updated should still return an empty body. Add another check if reponsecode is 204 than no body shall be returned.

**Review by ScorpioBroker — CHANGES_REQUESTED** [review]
Please also provide a test for this changes

**Review by vivekNEC — APPROVED** [review]

**Review by ScorpioBroker — APPROVED** [review]
LGTM

**Pull request merged**